doc/xmlto-config.xsl

```diff
@@ -1,6 +1,13 @@
 <?xml version='1.0'?>
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
                 xmlns:fo="http://www.w3.org/1999/XSL/Format"
+                exclude-result-prefixes="fo"
```

Comment:
Without this line, the generated HTML was:
```html
<meta xmlns:fo="http://www.w3.org/1999/XSL/Format" name="viewport" content="width=device-width, initial-scale=1.0"/>
```

Another option (which I _believe_ is fine, and probably cleaner) is to remove the `xmlns:fo` line together with this newly added one. I don't think that namespace is being used anywhere and should be useless.